starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,29 @@
-// Names and Input
+const hacker1 = "Benjamin"
+console.log(`The drivers name is ${hacker1}`)
 
+const hacker2 = "Alex"
+console.log(`The navigators name is ${hacker2}`)
 
-//Conditionals
+if(hacker1.length > hacker2.length){
+  console.log("The driver has the longer name, which is " + hacker1.length + " characters long")
+  }else if(hacker1.length < hacker2.length){
+    console.log("The navigator has the longer name " + hacker2.length + " characters long")
+  }else{
+    console.log("WOW! you both have equally long names " + hacker1.length + " characters long")
+  }
 
+let x = hacker1.toUpperCase();
+let y = x.split('');
+console.log(y.join(' '))
 
-// Lorem ipsum generator
+console.log(hacker2.split('').reverse().join(''))
+
+let total = [hacker1, hacker2];
+console.log(total.sort())
```

Comment:
this function should return only 1 value (the person who comes first).  One way to have done this is to do this `console.log(total.sort()[0])`. Another thing you could have done is use the localeCompare() function.